### ISSUE TITLE
Deactivate BIC

### ISSUE THREAD

**Issue by Detsieber** [Project60/org.project60.sepapp#15]
With the option "pp_hide_bic", the display of the BIC input field is disabled.
However, the BIC still needs to be processed in the background. And if the BIC related to the IBAN entered with the payment processor is not found (because it is a new one, that is not already downloaded), an error message is displayed and the BIC has to be entered manually.

Since the BIC is deprecated within SEPA rules already since several years, it would be desirable to get rid of that functionality:
We would like to be able to deactivate BIC processing completely, and would not need the littlebic extension anymore. This would result in having/needing no BIC in CiviSEPA anymore.

We should have another option (e.g. "pp_no_bic") that would trigger that behaviour.

**Comment by bjendres** [comment]
I think the original intention was to use this payment processor with a SEPA creditor that has BICs disabled. Does that still trigger the same behaviour?

**Comment by Detsieber** [comment]
Yes, even when BIC is disabled in the payment processor options and in the CiviSEPA option, the payment processor is "under the hood" still looking for a BIC... Which is not intended, I believe.
